### PR TITLE
Let a request handling delay be specified on the agent command line

### PR DESCRIPTION
**Pull request by rogerta**
This helps to simulate content analyses that can take longer time.

### CODE REVIEW

File: demo/agent.cc [hunk 1]
```diff
@@ -29,6 +31,11 @@ bool ParseCommandLine(int argc, char* argv[]) {
     if (arg.find(kArgUserSpecific) == 0) {
       path = kPathUser;
       user_specific = true;
+    } else if (arg.find(kArgDelaySpecific) == 0) {
+      delay = std::stoul(arg.substr(strlen(kArgDelaySpecific)));
+      if (delay > 30) {
```
Comment (nancy-lan-xiao) [inline demo/agent.cc:36]: Maybe you want to specify somewhere that the maximum delay is 30 seconds?

File: demo/agent.cc [hunk 2]
```diff
@@ -29,6 +31,11 @@ bool ParseCommandLine(int argc, char* argv[]) {
     if (arg.find(kArgUserSpecific) == 0) {
       path = kPathUser;
       user_specific = true;
+    } else if (arg.find(kArgDelaySpecific) == 0) {
```
Comment (nancy-lan-xiao) [inline demo/agent.cc:34]: To double check it is intentional: if you assign values to `path` & `user_specific`, you can't assign a value to `delay`?

File: demo/agent.cc (hunk 1 above)
Comment (rogerta) [reply demo/agent.cc:36]: Update the --help doc to state a max of 30 seconds.

File: demo/agent.cc (hunk 2 above)
Comment (rogerta) [reply demo/agent.cc:34]: I just tested and it is possible to set both.  This if / else if / else block is inside a fop loop that goes over al the args, so it should handle all of them.  Yes/no?

Comment (nancy-lan-xiao) [reply demo/agent.cc:34]: Makes sense, thanks for the explanation!

**Review by nancy-lan-xiao — APPROVED** [review]

**Pull request merged**